modules/core/src/main/resources/META-INF/classnames.properties

```diff
@@ -2083,6 +2083,8 @@ org.apache.ignite.internal.visor.baseline.VisorBaselineTaskArg
 org.apache.ignite.internal.visor.baseline.VisorBaselineTaskResult
 org.apache.ignite.internal.visor.baseline.VisorBaselineViewTask
 org.apache.ignite.internal.visor.baseline.VisorBaselineViewTask$VisorBaselineViewJob
+org.apache.ignite.internal.client.impl.id_and_tag.IdAndTagViewTaskResult
```

Comment:
Here you can see that you have to create a multiple unrelated packages with same name in different places. The idea of package is that it contains more than 1 thing instead of 2 packages containing just one thing.